programs/programs/wa/lifeline/calculator.py

```
@@ -0,0 +1,56 @@
+class WaLifeline(ProgramCalculator):
+    """Lifeline is a federal program providing a monthly discount on phone or internet service.
+    Eligible households qualify via income (≤135% FPL) or participation in a qualifying program
+    (Medicaid/Apple Health, SNAP, SSI, TANF, Section 8, or WIC). One benefit per household;
+    households already receiving Lifeline are excluded.
+
+    Data gaps: Pell Grant, Veterans Pension/Survivors Benefit, BIA General Assistance,
+    Tribal TANF, FDPIR, institutionalization status, and emancipated-minor exception.
+    """
+
+    fpl_percent = 1.35
+    # Standard federal benefit: $9.25/month → stored as annual value ($9.25 × 12 = $111)
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)